### PR TITLE
feat: new filter added

### LINKED ISSUE #4040: Create a booking status filter for Unconfirmed
Create a booking status filter for Unconfirmed

_Originally posted by @sean-brydon in https://github.com/calcom/cal.com/pull/4038#discussion_r958572460_

### PR DESCRIPTION
**Pull request by Udit-takkar**
Fixes: #4040 

Loom :- https://www.loom.com/share/bc3f92e625804979ad3de613247290ea

### CODE REVIEW

**Comment by Udit-takkar** [comment]
@sean-brydon i think it's working fine now.

File: packages/ui/v2/core/layouts/BookingLayout.tsx [hunk 1]
```diff
@@ -15,11 +15,11 @@ const tabs: (VerticalTabItemProps | HorizontalTabItemProps)[] = [
     icon: Icon.FiCalendar,
   },
   // TODO: Add filter for unconfimred bookings in a future PR - Out of scope
```
Comment (hariombalhara) [inline packages/ui/v2/core/layouts/BookingLayout.tsx:17]: The comment is stale now.

Comment (hariombalhara) [inline packages/ui/v2/core/layouts/BookingLayout.tsx:17]:
```suggestion
```

**Review by hariombalhara — APPROVED** [review]

**Review by hariombalhara — APPROVED** [review]

**Comment by hariombalhara** [comment]
Awesome work @Udit-takkar.

**Pull request merged**